server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -207,7 +206,23 @@ private Tag renderApplicantRow(Account applicant) {
             StyleUtils.even(Styles.BG_GRAY_100))
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant));
+        .with(renderActionsCell(applicant))
+        .with(renderDOBCell(applicant));
+  }
+
+  private Tag renderDOBCell(Account applicant) {
+
+    Optional<Applicant> newestApplicant = applicant.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
+    }
+    String applicantDOB = "";
+    Optional<String> dob = newestApplicant.get().getApplicantData().getDOB();
+    if (dob.isPresent()) {
+      applicantDOB = dob.get();
+    }
```

Comment:
suggestion (blocking): use `orElse` for accessing an `Optional` with a default value:
```suggestion
    String applicantDob = newestApplicant.get().getApplicantData().getDob().orElse("");
```